src/Ldap.php

```diff
@@ -1283,6 +1284,58 @@ public function delete($dn, $recursively = false)
     }
 
     /**
+     * Delete single attributes from a LDAP-Node
+     *
+     * This method removes single attributes from a node identified by
+     * <var>$dn</var>. The attributes have to be given as array where the
+     * array-key is the attribute-name and the array-value is the attribute-value
+     * that is to be removed.
+     *
+     * To remove multiple entries of an attribute pass an array with the values
+     * to be removed as value of the key. So if you want to remove more than
+     * one <strong>memberUid</strong>-attribute you would pass
+     * <code>array('memberUid' => ['uid1', 'uid2',...]);</code> as
+     * <var>$attributes</var>
+     *
+     * Beware that passing an empty array will remove <strong>all</strong> entries
+     * of the attribute. Therefore you will have to set the <var>$emptyAll</var>-flag!
+     *
+     * @param Dn|string $dn         The DN for which to remove attributes
+     * @param array     $attributes The attributes to be removed
+     * @param bool      $emptyAll   Whether empty attribute-array should remove
```

Comment:
I have some concerns about the argument name. May `allowEmptyAttributes` is better?